### ISSUE TITLE
Reactive Database

### ISSUE THREAD

**Issue by He-Pin** [FastTracks/TheAkkaWay#9]
Motivation:

integration with database query and updates with Akka & akka-stream

- slick and quill?
- jooq

**Comment by He-Pin** [comment]
@djx314 would you help to take care of this?

**Comment by He-Pin** [comment]
@jilen Would you help this out too?

**Comment by jilen** [comment]
@hepin1989  I would be interested  in this topic, but actually useful things are

+ How to query in a stream way, specially for very large dataset
+ How to turn subscription into `Stream` (Database like postgres support `Notify`).

But both topics seems a duplicate of #7

**Comment by djx314** [comment]
@hepin1989 For the database selection, I suggest that use `quill`.
I think slick's deadlock bug is still exsists sine somebody has submited some deadlock issue point to 3.2.0. It not the best time to use the slick in demo.
And quill can use the async database driver, I think it can bring better performance.

**Comment by He-Pin** [comment]
@djx314 @jilen Does `quill` current have `reactive stream` integration?
We plan first to translate the `RDP` book, and then come to this one.
https://github.com/ReactivePlatform/CodeSamples

**Comment by He-Pin** [comment]
@jilen Yes, they are related, so the final structure will need you fellows' review:)
